src/start/PasswordAuthView.js

```diff
@@ -20,6 +28,7 @@ class PasswordAuthView extends React.Component {
     email: string,
     password: string,
     error: string,
+    onepasswordHidden: boolean,
```

Comment:
`onepasswordHidden` -> `onePasswordHidden`

Also, I think it's clearer if you change this to `onePasswordSupported`